relayer/src/main.rs

```diff
@@ -348,3 +413,30 @@ fn create_eth_client(args: &EthereumArgs) -> EthApi {
     )
     .unwrap_or_else(|err| panic!("Error while creating ethereum client: {}", err))
 }
+
+async fn create_beacon_client(args: &BeaconRpcArgs) -> BeaconClient {
+    let timeout = args.beacon_timeout.map(Duration::from_secs);
+
+    BeaconClient::connect(args.beacon_endpoint.clone(), timeout)
+        .await
+        .expect("Failed to create beacon client")
+}
+
+fn decode_h256(hex: &str) -> H256 {
+    let data: [u8; 32] = decode_byte_array(hex);
+    data.into()
+}
+
+fn decode_h160(hex: &str) -> H160 {
+    let data: [u8; 20] = decode_byte_array(hex);
+    data.into()
+}
+
+fn decode_byte_array<const LEN: usize>(hex: &str) -> [u8; LEN] {
```

Comment:
This one clashes with `try_from_hex_encoded` from `ethereum_beacon_client` utils, isn't it?